src/registrar/forms/domain.py

```diff
@@ -51,9 +51,8 @@ def clean(self):
 
         ip_list = self.extract_ip_list(ip)
 
-        if ip and not server and ip_list:
-            self.add_error("server", NameserverError(code=nsErrorCodes.MISSING_HOST))
-        elif server:
+        # validate if the form has a server or an ip
+        if ip and ip_list or server:
```

Comment:
**suggestion (non-blocking)**: Could we parenthesize this conditional to make it more clear what the intention is? I think that Python's interpretation is going to be
```suggestion
        if (ip and ip_list) or server:
```
but I think it's easier to read if we don't have to remember the precedence rules.

Reply:
done